Assets/GLTF/Scripts/GLTFComponent.cs

```diff
@@ -18,9 +29,22 @@ class GLTFComponent : MonoBehaviour
 
 		IEnumerator Start()
 		{
+			/* prevent race condition with web server */
+			while (webServer == null) 
```

Comment:
Add a comment for why this code is waiting on webserver/expected scenario

Reply:
This was a problem with OSX and the build in web server to test the models. I recall one pull request fixing this issue, so it may not be longer necessary.